bundler/lib/bundler/rubygems_ext.rb

```diff
@@ -129,6 +129,34 @@ def to_lock
     end
   end
 
+  unless Gem::Requirement.new("> 1", "< 2") == Gem::Requirement.new("< 2", "> 1")
```

Comment:
Nice idea :+1:!

Reply:
A small little comment here. Feature detection is great, but I also somewhat like hardcoded version checks, because it makes it harder to miss future cleanup opportunities when dropping support for old versions.

So perhaps we can find a middle ground by adding a `# available since 3.2.0.rc.2` comment so that when a I grep for "3.2.0" in the future when dropping rubygems 3.2.0 support, I don't miss this.

Reply:
Extended to `# comparison is done order independently since rubygems 3.2.0.rc.2`, do you think it is better or not?

Reply:
Sure, yes!